packages/aws-client/tests/S3/test-multipartCopyObject.js

```diff
@@ -107,6 +135,7 @@ test("multipartCopyObject() sets the object's ACL", async (t) => {
     destinationBucket,
     destinationKey,
     ACL: 'public-read',
+    maxChunkSize: 5 * MB,
```

Comment:
do you need this?  Is it testing anything?

Reply:
removed